docs/CHANGELOG.rst

```diff
@@ -33,6 +33,9 @@ Changed
   problems parsing development version numbers
 - Support Slamdunk/Alleyoop processes in MultiQC
 - Enable sorting of files in ``alignment-star`` process using samtools
+- Add read length input to ``alleyoop-utr-rates`` and
```

Comment:
Changelog entry is not necessary for this commits, since the fix is for unreleased features which already have their own changelog entry lines.